packages/app/src/elements/item-view.ts

```diff
@@ -358,7 +371,13 @@ export class ItemView extends Routing(StateMixin(LitElement)) {
                 </header>
 
                 <pl-scroller class="stretch">
-                    <div class="vertical layout fill-vertically content">
+                    <div
```

Comment:
I decided to place this here and not support adding an attachment without a selected item via drag & drop, because it also requires adding a name, etc. It's possible, but the UX felt inferior, as if there's an item already selected.